packages/nuget/Sentry.AspNetCore/4.2.1.json

```diff
@@ -4,5 +4,30 @@
   "version": "4.2.1",
   "package_url": "https://www.nuget.org/packages/Sentry.AspNetCore",
   "repo_url": "https://github.com/getsentry/sentry-dotnet",
-  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnetcore"
+  "main_docs_url": "https://docs.sentry.io/platforms/dotnet/aspnetcore",
+  "categories": [
+    "server"
+  ],
+  "features": {
+    "crons": {
+      "availability": "GA",
+      "available-with-version": "4.2.0"
+    },
+    "errors": {
+      "availability": "GA",
+      "available-with-version": "1.0.0"
+    },
+    "performance": {
+      "availability": "GA",
+      "available-with-version": "3.0.0"
+    },
+    "profiling": {
+      "availability": "GA",
```

Comment:
oh profiling isn't GA. even though we messed up the package version and removed the -beta